src/components/plugin-app.tsx

```diff
@@ -31,15 +31,18 @@ export default class PluginApp extends React.Component<IProps, IState> {
   public render() {
     const { type } = this.props;
     switch (type) {
-      case "windowShade": return this.renderWindowShade();
+      case "windowShade":
+      case "teacherTip":
+      case "theoryAndBackground":
+        return this.renderWindowShade();
```

Comment:
I get the idea. I'm wondering whether we should use this approach or "type": "windowShade", and then use another property to define style, like "style": "teacherTip"/"theoryAndBackground", etc. Not sure now, but I'd keep in in mind when we implement more types and authoring, as it might get more important then. Authoring could hide this from user anyway.

Reply:
I like this idea. I'm not sure when it will be best to make this change, but we should do it. Maybe after these pending PR's are merged.